### PR TITLE
feat(workflow): AlertRule 'environment' is_dangerous migration

### PR DESCRIPTION
**Pull request by iProgramStuff**
This is the dangerous migration for the AlertRule environment field addition. Going to manually generate + run DDL in cooperation with an ops member.

### CODE REVIEW

File: src/sentry/migrations/0027_alert_rule_environment.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Adding indexes to large tables. These indexes should be created concurrently,
+    #   unfortunately we can't run migrations outside of a transaction until Django
+    #   1.10. So until then these should be run manually.
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = True
```
Comment (markstory) [inline src/sentry/migrations/0027_alert_rule_environment.py]: The `alertrule` table is pretty small < 100 rows, is there something else that is making this need to be offline?

Comment (wedamija) [reply src/sentry/migrations/0027_alert_rule_environment.py]: We weren't sure how risky adding an FK to `environment` would be, so are doing it offline.

**Review by wedamija — COMMENTED** [review]
Looks good, although you'll need to either regenerate this or renumber/update the dependent migration, since someone has merged a new one ahead of you

**Review by wedamija — APPROVED** [review]

File: src/sentry/migrations/0027_alert_rule_environment.py (hunk 1 above)
Comment (markstory) [reply src/sentry/migrations/0027_alert_rule_environment.py]: That makes sense.

**Pull request closed without merging**